core/src/avm2/globals/String.as

```diff
@@ -53,9 +53,9 @@ package {
                 return s.AS3::slice(start, end);
             };
 
-            prototype.split = function(delimeter:* = void 0, limit:* = 4294967295.0):Array {
+            prototype.split = function(delimiter:* = void 0, limit:* = 4294967295.0):Array {
                 var s:String = this;
-                return s.AS3::split(delimeter, limit);
+                return s.AS3::split(delimiter, limit);
```

Comment:
https://docs.ruffle.rs/en_US/FlashPlatform/reference/actionscript/3/String.html#split()

Not aware of a misspelling in actual FP (like `imputVector` of `BitmapData.setVector`).